hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1295,6 +1295,9 @@
   /** Configuration key for SplitLog manager timeout */
   public static final String HBASE_SPLITLOG_MANAGER_TIMEOUT = "hbase.splitlog.manager.timeout";
 
+  /** To allow the empty entries to get filtered  which have no cells or all cells got filtered though WALCellFilter */
+  public static final String HBASE_REPLICATION_WAL_FILTER_EMPTY_ENTRY = "hbase.replication.wal.filteremptyentry";
```

Comment:
This doesn't belong here. You are already changing BaseReplicationEndpoint, put it there?

Reply:
@sandeepvinayak  Do you plan addressing this comment here? I just saw this before I tried to merge this. Thanks.